packages/react-server-website/middleware/Theme/button.less

```diff
@@ -7,7 +7,7 @@ button {
 .Button {
 	display: block;
 	text-decoration: none;
-	color: #FFFFFF;
+	color: white;
```

Comment:
Was this change just a personal preference of yours or is there an additional benefit I don't know about?